charts/graphprotocol-node/templates/deployment.yaml

```diff
@@ -34,6 +34,13 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
+        - name: wait-for-rta
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
I think we could use more lightweight image, wdyt?

Reply:
We are using here same image. If we would use some lightweight image we would be pulling 2 different images wich would be less "lightweight" than using single image. :)